multiversx_sdk_cli/accounts.py

```diff
@@ -42,7 +43,10 @@ def __init__(self,
             self.secret_key = user_pem_file.secret_key.hex()
             self.address = Address(user_pem_file.public_key.hex())
         elif key_file and password:
-            address_from_key_file, secret_key = load_from_key_file(key_file, password)
+            with open(key_file, 'r') as f:
+                key_file_data = load(f)
```

Comment:
Usually, we do `json.load`, instead of `from json import load` (a bit more explicitly).